.github/workflows/databend-release.yml

```
@@ -169,6 +146,17 @@ jobs:
           cp ./target/aarch64-unknown-linux-gnu/release/databend-meta ./distro/linux/arm64
           cp ./target/aarch64-unknown-linux-gnu/release/databend-query ./distro/linux/arm64
 
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_NAMESPACE }}/databend:${{ steps.get_version.outputs.VERSION }}
+          platforms: linux/amd64
+          context: .
+          file: ./docker/Dockerfile
+          build-args: VERSION=${{ steps.get_version.outputs.VERSION }}
```

```suggestion
          file: ./docker/Dockerfile
```

I think it has the same functionality as `file: ./docker/release/Dockerfile`. However, the `build-args` should be removed, as it is useless.